File: .sdk/tm/dart/test/exists_test.dart
import 'harness.dart';

import '../lib/SolardemoSDK.dart';

void tests() {
  describe('exists', () {
    test('test-mode', (t) async {
      final testsdk = SolardemoSDK.test();
      equal(true, null != testsdk);
    });
  });
}

File: .sdk/tm/js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { SolardemoSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await SolardemoSDK.test()
    equal(null !== testsdk, true)
  })

})
